src/sentry/processing/backpressure/health.py

```diff
@@ -68,6 +68,15 @@ def is_consumer_healthy(consumer_name: str = "default") -> bool:
                     "reason": reason,
                 },
             )
+            metrics.event(
+                f"{consumer_name}: backpressure.consumer.unhealthy",
```

Comment:
we might not want to include the consumer name in the event, we've got it as a tag and that might make it harder to combine the events for analysis, overlays, etc. Might be worth experimenting with manually to validate what we want name, tags, etc. set as based on the data that we have when we generate the event.